homeassistant/components/elmax/common.py

```diff
@@ -62,11 +64,17 @@ def panel_status(self) -> PanelStatus | None:
         """Return the last fetched panel status."""
         return self.data
 
-    def get_endpoint_state(self, endpoint_id: str) -> DeviceEndpoint | None:
+    def get_endpoint_state(self, endpoint_id: str) -> DeviceEndpoint:
```

Comment:
It was previously used before the refactor. It has now been removed thanks to your suggestion. It will be probably needed later on as we will add more Elmax platforms.